src/quicklist.c

```diff
@@ -173,8 +179,13 @@ quicklist *quicklistCreate(void) {
     return quicklist;
 }
 
+/* quicklist 最大压缩级别 （2**16-1）*/
```

Comment:
这可能要提一下 x64，32 bits 的话这个值我记得是不一样的
```suggestion
/* quicklist 最大压缩级别 （64 位操作系统下：2**16-1）*/
```